jenkins/bootstrap_test.py

```diff
@@ -1527,6 +1581,19 @@ def testNoBadVarsInJobs(self):
             if bad_vars:
                 self.fail('Job %s contains bad bash variables: %s' % (job, ' '.join(bad_vars)))
 
+    def testValidJobEnvs(self):
+        """Validate jobs/config.json."""
+        with open(bootstrap.test_infra('jobs/config.json')) as fp:
+            config = json.loads(fp.read())
+            for job in config:
+                self.assertTrue('scenario' in config[job], job)
+                self.assertTrue(os.path.isfile(bootstrap.test_infra('scenarios/%s.py' % config[job]['scenario'])), job)
+                if 'args' in config[job]:
+                    for arg in config[job].get('args', []):
+                        m = re.match(r'--env-file=([^\"]+)', arg)
```

Comment:
If the scenario is kubernetes-e2e I would validate that there is at least one --env-file flag